src/main/resources/view/PersonListCard.fxml

```diff
@@ -7,32 +7,62 @@
 <?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
 
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
   <GridPane HBox.hgrow="ALWAYS">
     <columnConstraints>
       <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
     </columnConstraints>
     <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="0">
       <padding>
-        <Insets top="5" right="5" bottom="5" left="15" />
+        <Insets bottom="5" left="15" right="5" top="5" />
       </padding>
-      <HBox spacing="5" alignment="CENTER_LEFT">
+      <HBox alignment="CENTER_LEFT" spacing="5">
         <Label fx:id="id" styleClass="cell_big_label">
           <minWidth>
             <!-- Ensures that the label text is never truncated -->
             <Region fx:constant="USE_PREF_SIZE" />
           </minWidth>
         </Label>
-        <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
+        <Label fx:id="name" styleClass="cell_big_label" text="\$first" />
       </HBox>
       <FlowPane fx:id="tags" />
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
-      <Label style="-fx-font-weight: bold;" styleClass="cell_small_label" text="${'\nNotes:'}" />
-      <Label fx:id="notes" styleClass="cell_small_label" text="\$notes" />
+         <HBox alignment="CENTER_LEFT" spacing="5">
+            <children>
+               <VBox maxWidth="300.0">
+                  <children>
+                  <Label style="-fx-font-weight: bold;" styleClass="cell_small_label" text="${'\nStrengths:'}" />
+                  <Label fx:id="strengths" maxWidth="150.0" style="-fx-background-color: #fff2cd; -fx-text-fill: #000000;" styleClass="cell_small_label" text="\$strengths" wrapText="true" />
```

Comment:
I think maybe it is better to write style things in a separate `.css` file but this one is also acceptable

Reply:
I agree with XiaoTeng on this. I think it's a good idea to have styles written in a separate CSS file

Reply:
I'll create a separate `.css` file for these lines. Thanks for letting me know 👍